manifests/config.pp

```diff
@@ -47,12 +47,10 @@
     }
     'Debian': {
       # Remove debian defaults conf
-      if $facts['os']['release']['major'] == '16.04' {
-        file { 'defaults-debian.conf':
-          ensure  => absent,
-          path    => "${::fail2ban::config_dir_path}/jail.d/defaults-debian.conf",
-          require => $::fail2ban::config_file_require,
-        }
+      file { 'defaults-debian.conf':
+        ensure  => absent,
+        path    => "${::fail2ban::config_dir_path}/jail.d/defaults-debian.conf",
```

Comment:
please don't add the leading ::. they are not needed anymore since we don't support Puppet 3.

Reply:
done